### DISCUSSION TITLE
Cylindric Custom Support

### DISCUSSION THREAD

**Discussion by sdauer** [5axes/CustomSupportCylinder#39] [General]
Hello
We've been using the great Cylindric Custom Support plugin for several months, but the latest Cura update has made your plugin unavailable...
Are you planning to update it to versions 5.7?
Thank you for your help.
Sincerely,
Sebastian

**Comment by 5axes** [comment]
There was a bug in Cura . But it had been solved in Cura 5.7.2 but a New one is now present. I m waiting for a correction from Ultimaker and then the plugin will be available in the marketplace. Until this date you can install the actual release from the release present on this Github.

**Reply by sdauer** [reply]
Oh thank you for your prompt feedback.
I'm already in cura 5.7.2 but I don't see anything actually.
I can't download the current version on Github, the site tells me "The given author does not exist."
Can you help me? I would understand that you do not have time.
Thank you.
Sincerely,
Sebastian

**Reply by petelton** [reply]
Hello.
I am facing the same issue like you..
Form Ultimaker Cura 5.6 release I can't download "cylindric custom support" because isn't available from marketplace to install.
How can I fix it this?
Thanks

**Comment by sdauer** [comment]
Hello
As indicated by the developer who kindly answered us, we have to wait for the Ultimaker fix for the plugin to be available.
We have no other solutions to our knowledge!
Have a nice day.
